example-shelley/cardano-ledger-example-shelley.cabal

```diff
@@ -0,0 +1,141 @@
+cabal-version:       2.2
+
+name:                cardano-ledger-example-shelley
+version:             0.1.0.0
+synopsis:            Example era within Shelley context
+description:
+  This is intended as a demonstration and example for hard fork
+  combinators.
+bug-reports:         https://github.com/input-output-hk/cardano-ledger-specs/issues
+license:             Apache-2.0
+author:              IOHK Formal Methods Team
+maintainer:          formal.methods@iohk.io
+copyright:           2020 Input Output (Hong Kong) Ltd.
+category:            Network
+build-type:          Simple
+
+source-repository head
+  type:     git
+  location: https://github.com/input-output-hk/cardano-ledger-specs
+
+library
+  exposed-modules:
+    Cardano.Ledger.Example
+    Cardano.Ledger.Example.Translation
+
+  -- other-extensions:
+  build-depends:
+    array,
+    base >=4.9 && <4.15,
+    bytestring,
+    cardano-binary,
+    cardano-crypto-class,
+    cardano-ledger-core,
+    cardano-prelude,
+    cardano-slotting,
+    cborg,
+    containers,
+    data-default-class,
+    deepseq,
+    groups,
+    mtl,
+    nothunks,
+    prettyprinter,
+    primitive,
+    shelley-spec-ledger,
+    small-steps,
+    text,
+    transformers
+  hs-source-dirs: src
+  ghc-options:
+    -Wall
+    -Werror
+    -Wcompat
+    -Wincomplete-record-updates
+    -Wincomplete-uni-patterns
+    -Wredundant-constraints
+    -Wpartial-fields
+    -Wunused-imports
+    -Wdodgy-imports
+  default-language:    Haskell2010
+
+library test
+  exposed-modules:
+    Test.Cardano.Ledger.Example
+  -- other-extensions:
+  build-depends:
+    array,
+    cardano-ledger-example-shelley,
+    shelley-spec-ledger-test,
+    base >=4.9 && <4.15,
+    bytestring,
+    cardano-binary,
+    cardano-crypto-class,
+    cardano-crypto-praos,
+    cardano-prelude,
+    generic-random,
+    cardano-slotting,
+    cborg,
+    containers,
+    hashable,
+    mtl,
+    shelley-spec-ledger,
+    small-steps,
+    tasty-hunit,
+    tasty-quickcheck,
+    tasty,
+    text,
+    QuickCheck >= 2.13.2,
+  hs-source-dirs: src
```

Comment:
Should this not be `test`?

Reply:
OK

Reply:
Wow I fat fingered this reorganization. Surprised my manual tests didn't catch it, but CI sure did!